Cost Estimation/BrickWallCostEstimate.FCMacro

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+__Name__ = 'CostEstimate'
+__Comment__ = 'Estimates brick, cement, sand cost of all objects that have IFCRole=Wall'
+__Author__ = 'haseeb'
+__Version__ = '0.1'
+__Date__ = '2019-6-24'
+__License__ = 'LGPL-3.0-or-later'
+__Web__ = ''
+__Wiki__ = ''
+__Icon__ = ''
+__Help__ = ''
+__Status__ = 'Alpha'
+__Requires__ = 'FreeCAD >= v0.17'
+__Communication__ = ''
+__Files__ = ''
+
+import FreeCAD, math
```

Comment:
Please use import `FreeCAD as app` and use `app` everywhere instead of FreeCAD.
Also add `import FreeCADGui as gui`.

Please organize your import (standard lib, then system libs then "project libs"), one per line.